ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -29,6 +29,7 @@
 #include <unistd.h>
 #include <limits.h>
 #include <algorithm>
+#include <thread>
```

Comment:
Do you really need this?

Reply:
it is necessary to use std::thread::hardware_concurrency()